src/main/kotlin/Main.kt

```diff
@@ -1,7 +1,14 @@
-fun main(args: Array<String>) {
-    println("Hello World!")
+fun main() {
+    val intArray: Array<Int> = arrayOf(6,9,24)
```

Comment:
In Kotlin we have "Lists" which are more powerful than arrays. Here is link where people discuss the difference between them.
https://stackoverflow.com/questions/36262305/difference-between-list-and-array-types-in-kotlin

Reply:
Thabk you I will look intro it.